EduVPN/Services/ConnectionService.swift

```diff
@@ -422,33 +423,41 @@ private extension ConnectionService {
 
         var configBuilder = parseResult.configuration.builder()
         configBuilder.tlsSecurityLevel = 3 // See https://github.com/eduvpn/apple/issues/89
+        let config = configBuilder.build()
 
-        var providerConfigBuilder = OpenVPNProvider.ConfigurationBuilder(sessionConfiguration: configBuilder.build())
-        providerConfigBuilder.masksPrivateData = false
-        providerConfigBuilder.shouldDebug = true
+        var providerConfig = OpenVPN.ProviderConfiguration(Config.shared.appName, appGroup: appGroup, configuration: config)
+        providerConfig.masksPrivateData = false
+        providerConfig.shouldDebug = true
```

Comment:
Is this on on purpose? Should not debug on production, I suppose.

Reply:
Yes, `shouldDebug` is set to true on purpose.

It was turned on in #274 because at that time, that was how we could get TunnelKit to write the log to disk (see [urlForLog at that point](https://github.com/eduvpn/apple/blob/57396b0ee0823dbd7e212356d07d3babf6ae0170/Pods/TunnelKit/TunnelKit/Sources/Protocols/OpenVPN/AppExtension/OpenVPNTunnelProvider%2BConfiguration.swift#L246)).

With this PR, we do the writing to disk in our own code, but we still would want to keep the log level as `.debug` because:
 - it adds useful information to the log
 - it enables us to see the log of the production app in the Console.app

Reply:
As it turns out, `shouldDebug` isn't actually required anymore. Good catch, @johankool.